### ISSUE TITLE
remove-hpovNetwork 

### ISSUE THREAD

**Issue by scriptengine** [HewlettPackard/POSH-HPOneView#176]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
remove-hpovNetwork not to throw an error when supplied with a network name that has been validated with get-hpovnetwork

### Actual Behavior
PS> remove-HPOVNetwork Management-100 -Confirm:$false
Send-HPOVRequest : Cannot validate argument on parameter 'uri'. -URI must 
being with a '/' (eg. /rest/server-hardware) in its value. Please correct the 
value and try again.
At C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\HPOneView.200\HPOneView.2
00.psm1:36649 char:33
+                     $net = Send-HPOVRequest $net -ApplianceConnection ...
+                                             ~~~~
    + CategoryInfo          : InvalidData: (:) [Send-HPOVRequest], ParameterBi 
   ndingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Send-HPOVReques 
   t
 
remove-HPOVNetwork : Cannot validate argument on parameter 'uri'. -URI must 
being with a '/' (eg. /rest/server-hardware) in its value. Please correct the 
value and try again.
At line:1 char:1
+ remove-HPOVNetwork Management-100 -Confirm:$false
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Remove-HPOVNetwork], Parameter 
   BindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Remove-HPOVNetw 
   ork
 

### Steps to reproduce
$aNetwork = (Get-HPOVNetwork | select -first 1).name

#Demostrate network name is valid
Get-HPOVNetwork -name $aNetwork | ft name,type,purpose,"Vlanid"

#Repo issue
Remove-HPOVNetwork -name $aNetwork -Confirm:$false

### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):
PowerShell Library Version : 2.0.621.0
Library Path               : C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
                             \HPOneView.200

HPE OneView Appliance Version:  
 2.00.02-0219211

Output from `$PSVersionTable` on your Windows Host:
Name                           Value                                           
----                           -----                                           
PSVersion                      5.1.14393.206                                   
PSEdition                      Desktop                                         
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                         
BuildVersion                   10.0.14393.206                                  
CLRVersion                     4.0.30319.42000                                 
WSManStackVersion              3.0                                             
PSRemotingProtocolVersion      2.3                                             
SerializationVersion           1.1.0.1

**Comment by ChrisLynchHPE** [comment]
Update your script to perform the following operation instead:

```PowerShell
PS C:\> $aNetwork = Get-HPOVNetwork | select -first 1
PS C:\> Remove-HPOVNetwork -Resource $aNetwork -Confirm:$false
```

Please know that future PowerShell library versions will require objects, and not resource names for non-Get Cmdlets.

**Comment by scriptengine** [comment]
Chris: Thanks for the suggestions. I have it working using something similar.
Thanks also for the pointer that nonGet cmlets use objects. I'll start from that point each time.

Can I suggest the help is updated so others don't fall into the same hole as I did trying to use a [string]/name

From help remove-HPOVnetwork
"-Resource <Object>
    Aliases [-**name**, -Network]

    The network object(s) or **name(s)** to be deleted."
(the use of "name in the above")

and
"INPUTS
    System.String
        **Network Name** or URI"
Again the use of a network name is documented

**Comment by ChrisLynchHPE** [comment]
There actually was a slight bug in the Cmdlet.  The 2.00 Cmdlets still do support Resource Names, so the documentation for 2.00 Cmdlets does not need to change.  I'll be posting an updated 2.00 library that will fix this.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 2.0.630.0](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v2.0.630.0).

Closing
